inference-engine/src/transformations/include/transformations/serialize.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+// ATTENTION :
+// - dynamic shapes are not supported
+// - GenericIE operation type (experimental opset) is not supported
+// - order of generated layers in xml file is ngraph specific (given by
+// get_ordered_ops()); MO generates file with different order, but they are
+// logically equivalent
+
+#pragma once
+
+#include <string>
+
+#include "ngraph/pass/pass.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API Serialize;
+
+}  // namespace pass
+}  // namespace ngraph
+
+// ! [function_pass:serialize_hpp]
```

Comment:
Do we insert this code snippet somewhere in documentation? If no let's remove this doxygen tags.

Reply:
We don't. I just copy-pasted from template code, and was not sure if it's needed. Thanks for pointing this out, I'll remove.

Reply:
Done in this [commit](https://github.com/openvinotoolkit/openvino/pull/2579/commits/1be8c85bc06795554e80cf5b9903214b2f2248e7)